app/view/fund_transfer.py

```diff
@@ -29,7 +29,16 @@ def post(self):
             db.session.commit()
             output = fund_transfer_schema.dump(fund)
             from_account = BankAccount.query.filter(BankAccount.account_number == fund_data['from_account']).first()
+            if not from_account:
+                response = ResponseGenerator(data={}, message="Invalid Account number, from_account", success=False,
+                                             status=status.HTTP_400_BAD_REQUEST)
+                return response.error_response()
             to_account = BankAccount.query.filter(BankAccount.account_number == fund_data['to_account']).first()
+            if not to_account:
+                response = ResponseGenerator(data={}, message="Invalid Account number, to_account",
+                                             success=False,
+                                             status=status.HTTP_400_BAD_REQUEST)
+                return response.error_response()
             transaction_type = TransactionType.query.filter(TransactionType.transaction_type == "debit").first()
```

Comment:
Use constants for `debit` and `credit` and use those constants at all the places where you have used `debit` and `credit`

Reply:
Have taken transaction_type_1 and transaction_type_2 as debit and credit constants and used at all the places required.